src/lodashMin.ts

```diff
@@ -0,0 +1,36 @@
+/**
+* lodashMini.ts
```

Comment:
Please rename this file to lodashMini to stay in line with convention in other projects and the name specified in this comment
See as reference: https://github.com/Microsoft/reactxp/blob/master/src/common/lodashMini.ts